### PR TITLE
Add member modal bug fixed

### PR DESCRIPTION
**Pull request by NitinCopods**

### CODE REVIEW

File: app/components/members/AddMemberModal.tsx [hunk 1]
```diff
@@ -75,7 +75,8 @@ export default function AddMemberModal({
                         type="text"
                         name="firstName"
                         className=" my-1.5 h-11 w-full rounded-lg border border-gray-200 px-3 text-base"
```
Comment (yash6102000) [inline app/components/members/AddMemberModal.tsx:77]: remove extra spacing

File: app/components/sections/QuestionCard.tsx [hunk 2]
```diff
@@ -41,22 +41,7 @@ const QuestionCard = ({
...
-                isExpanded == -1 ? index : isExpanded == index ? -1 : index
-              )
-          }}
-          onClick={() => {
-            onAccordianToggle(
-              isExpanded == -1 ? index : isExpanded == index ? -1 : index
-            )
-          }}
-        >
+        <div className="items-top flex flex-1 items-start justify-between">
```
Comment (yash6102000) [inline app/components/sections/QuestionCard.tsx:44]: remove items top

File: app/components/sections/SectionDetails.tsx [hunk 3]
```diff
@@ -60,6 +60,8 @@ const SectionDetails = () => {
             .includes(searchText.toLowerCase())
         })
         .map((question: Question, i: number) => {
+          console.log(i)
```
Comment (arpita-20) [inline app/components/sections/SectionDetails.tsx:63]: remove console.log

File: app/components/sections/QuestionCard.tsx [hunk 4]
```diff
@@ -41,22 +41,7 @@ const QuestionCard = ({
...
     >
       <div className="items-top break flex justify-between text-base text-gray-600">
-        <div
-          className="items-top flex flex-1 items-start justify-between"
-          tabIndex={0}
-          role="button"
-          onKeyUp={(e) => {
-            if (e.key === 'Enter')
-              onAccordianToggle(
-                isExpanded == -1 ? index : isExpanded == index ? -1 : index
```
Comment (arpita-20) [inline app/components/sections/QuestionCard.tsx:51]: why are you removing this part. could you please explain on comments

File: app/components/members/AddMemberModal.tsx [hunk 5]
```diff
@@ -15,6 +17,9 @@ export default function AddMemberModal({
   setOpen: (e: boolean) => void
 }) {
   const transition = useTransition()
+  const [onFirstName, setOnFirstName] = useState('')
```
Comment (anuragP04) [inline app/components/members/AddMemberModal.tsx:20]: variable names are not relevant, onFirstName?

File: app/components/members/AddMemberModal.tsx [hunk 6]
```diff
@@ -58,7 +63,7 @@ export default function AddMemberModal({
                     </h2>
                     <Icon
                       tabIndex={0}
-                      className="cursor-pointer text-2xl text-gray-600"
+                      className="cursor-pointer text-2xl text-gray-600 focus:outline-none"
```
Comment (anuragP04) [inline app/components/members/AddMemberModal.tsx:66]: why have you set outline to none

File: app/components/DeletePopUp.tsx [hunk 7]
```diff
@@ -55,7 +55,7 @@ export default function DeletePopUp({
                     <div className="mx-auto flex h-12 w-12 flex-shrink-0 items-center justify-center rounded-full bg-red-100 sm:mx-0 sm:h-10 sm:w-10">
                       <Icon
                         icon="ic:outline-delete-outline"
-                        className="h-6 w-6 text-red-500"
+                        className="h-6 w-6 text-red-500 focus:outline-none"
```
Comment (anuragP04) [inline app/components/DeletePopUp.tsx:58]: why have you set outline to none?

File: app/components/sections/AddSection.tsx [hunk 8]
```diff
@@ -17,6 +18,8 @@ const AddSection = ({
   showErrorMessage: boolean
 }) => {
   const transition = useTransition()
+  const [onAddSectionName, setOnAddSectionName] = useState('')
```
Comment (anuragP04) [inline app/components/sections/AddSection.tsx:21]: variable names are not relevant

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**